### ISSUE TITLE
Port core quickstart code to use cookiecutter?

### ISSUE THREAD

**Issue by nixjdm** [lektor/lektor#647]
It has been suggested to me by another dev that we could refactor the quickstart logic to use [cookiecutter](https://github.com/audreyr/cookiecutter). This has advantages of trimming the our codebase for something pretty widely used. I am unsure of potential disadvatages at this time. I haven't used cookiecutter much. I'm not particularly worried about the additional dependency, as it is a pretty widely used and stable project as far as I can tell.

Does cookiecutter limit us in any important ways? How might it impact #646? Would it easily enough integrate into the rest of our CLI?

**Comment by skorokithakis** [comment]
I used cookiecutter a few times, but not enough that I'm particularly knowledgeable on it. It worked quite well, it basically renders a template where the template is the entire project. I don't know if it can support running commands afterwards, but the quickstart could always just call cookiecutter as a first step and then continue if cookiecutter succeeds...

**Comment by runfalk** [comment]
Looking at the number of PRs that haven't received a single comment, the project looks a bit stagnated. That's the only thing that makes me a bit cautious.

**Comment by andoresuperesu** [comment]
I agree with @rubfalk I've got work under construction trying to contribute, but time is limited. Could we do something like a feature lock and focus on a few or single thing?

**Comment by runfalk** [comment]
@andresperezcera I was thinking of the cookie cutter project, but I guess it applies to Lektor in some ways too :sweat_smile:

**Comment by andoresuperesu** [comment]
@runfalk :sweat_smile: Hehe, yeah... Cookiecutter... That's what I meant, sure. Anyways, I understood it in reference to our ability to manage testing of yet another solution, I'm stuck with limited time to work on the windows side of things (which seems much more fragile). However, what would this solve or improve?
